### PR TITLE
using artipie parent pom

### PR DESCRIPTION
**Pull request by g4s8**
Using artipie parent pom, removed dependency and plugin configurations

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#51](https://github.com/artipie/docker-adapter/issues/51) is assigned to @iakunin[/z](https://www.0crat.com/u/iakunin), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/d143ebb6-7a04-43ed-8ca6-1069f1b9fed1); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: pom.xml [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
-    <junit-platform.version>5.4.2</junit-platform.version>
-  </properties>
   <licenses>
     <license>
       <name>MIT License</name>
-      <url>https://github.com/artipie/docker-adapter/blob/master/LICENSE</url>
+      <url>https://github.com/artipie/docker-adapter/blob/master/LICENSE.txt</url>
```
Comment (iakunin) [inline pom.xml]: @g4s8 seems that this link is broken: there is no `LICENSE.txt` in master branch, but only just `LICENSE`

File: pom.xml [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (iakunin) [inline pom.xml]: @g4s8 maybe we can break this line into multiple ones?

File: pom.xml (hunk 1 above)
Comment (g4s8) [reply pom.xml]: @iakunin it will change in this commit: license was renamed from `LICENSE` to `LICENSE.txt`

File: pom.xml (hunk 2 above)
Comment (g4s8) [reply pom.xml]: @iakunin it's not possible because of `xcop` linter error

File: pom.xml (hunk 1 above)
Comment (iakunin) [reply pom.xml]: @g4s8 right: just missed this renaming in the PR.

File: pom.xml (hunk 2 above)
Comment (iakunin) [reply pom.xml]: @g4s8 got it

**Comment by yegor256** [comment]
@g4s8 thanks!

**Comment by iakunin** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/docker-adapter/issues/51#issuecomment-582921210))

@iakunin This is what I know about this job in [CT2E6TK9B](https://www.0crat.com/p/CT2E6TK9B), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#51](https://github.com/artipie/docker-adapter/issues/51) is [in scope](http://www.zerocracy.com/policy.html#14) for 23hr
  * The role is `REV`
  * The job is assigned to @iakunin[/z](https://www.0crat.com/u/iakunin) for [20hr](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @iakunin[/z](https://www.0crat.com/u/iakunin) will get 25% of their hourly rate for this job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @g4s8[/z](https://www.0crat.com/u/g4s8): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fdocker-adapter%2351%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by iakunin** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/docker-adapter/issues/51#issuecomment-583258532))

@iakunin This is what I know about this job in [CT2E6TK9B](https://www.0crat.com/p/CT2E6TK9B), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#51](https://github.com/artipie/docker-adapter/issues/51) is [in scope](http://www.zerocracy.com/policy.html#14) for 2days
  * The role is `REV`
  * The job is assigned to @iakunin[/z](https://www.0crat.com/u/iakunin) for [2days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @iakunin[/z](https://www.0crat.com/u/iakunin) will get 25% of their hourly rate for this job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @g4s8[/z](https://www.0crat.com/u/g4s8): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fdocker-adapter%2351%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by iakunin** [comment]
@g4s8 is it ok that PR is merged, but @0crat didn't closed the job?

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/artipie/docker-adapter/issues/51#issuecomment-583262877); you must always start your message with my name if you want to address it to me, see [§1](http://www.zerocracy.com/policy.html#1)

**Comment by g4s8** [comment]
@0crat out

**Comment by 0crat** [comment]
Job was finished in 38 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @iakunin[/z](https://www.0crat.com/u/iakunin)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**